Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-slim
+
+ADD . /app
+RUN pip install -r /app/requirements.txt &&\
+    pip install -e /app
+
+CMD ["crypt4gh"]
```

Comment:
Not sure that's interesting to have it here.

Reply:
I'd like to use this for automated building and testing. This will also be useful for deployment. Happy to chat. :)

Reply:
That's exactly my point: You should keep this in your own deployment system. Not here.
In your deployment, you need to build the docker image. We don't use the CI or some Github/DockerHub hooks for that.

This only packages the code in a docker image, and calls `crypt4gh` if nothing else is specified when calling `docker run`.
Moreover, for your deployment, maybe it's better to use `ENTRYPOINT` rather than `CMD` so that you can pass arguments in the `docker run` line.

Reply:
It's just a way to run it in a repeatable fashion. This could also be used for automated testing. I like your suggestion for ENTRYPOINT-- can swap that over.

Reply:
True. Let's wait for @blankdots to finish the tests, and he could put that file in the `tests` directory (instead of the repository's root).